products/accesscontextmanager/api.yaml

```diff
@@ -66,3 +66,200 @@ objects:
         description: |
           Time the AccessPolicy was updated in UTC.
         output: true
+  - !ruby/object:Api::Resource
+    name: 'AccessLevel'
+    # This is a weird API, so we need to use a few fields to map the methods to
```

Comment:
nit: I'd replace the word 'weird' with something less negative, like 'unusual' or 'atypical'.